scripts/Makefile.build

```diff
@@ -262,7 +262,7 @@ $(obj)/%.lst: $(src)/%.c FORCE
 # Compile Rust sources (.rs)
 # ---------------------------------------------------------------------------
 
-rust_allowed_features := new_uninit
+rust_allowed_features := new_uninit,impl_trait_in_assoc_type
```

Comment:
Will this actually be stabilized soon? If not, then I would prefer to declare a struct and implement the traits there rather than doing a type alias impl trait.

Reply:
`return_position_impl_trait_in_trait` just got into FCP, which means it may be available in 1.76.0 if all goes well.

Reply:
It was merged earlier today https://github.com/rust-lang/rust/pull/115822

Reply:
1.75.0, even better.